### PR TITLE
feat: add graphql resolver for adding spans to datasets

### LINKED ISSUE #3178: [datasets][gql] addSpansToDataset

### PR DESCRIPTION
**Pull request by axiomofjoy**
Adds `addsSpansToDataset` GraphQL resolver. I'm going to refactor our mutations into separate modules after landing this PR.

resolves #3178

### CODE REVIEW

File: src/phoenix/server/api/schema.py [hunk 1]
```diff
@@ -384,6 +402,140 @@ async def create_dataset(
...
+                await session.execute(
+                    insert(models.DatasetExampleRevision).values(
+                        dataset_example_id=dataset_example_rowid,
+                        dataset_version_id=dataset_version_rowid,
+                        input=get_dataset_example_input(span),
+                        output=get_dataset_example_output(span),
+                        metadata_=span.attributes,
+                        revision_kind="CREATE",
+                    )
+                )
```
Comment (axiomofjoy) [inline src/phoenix/server/api/schema.py:505]: Is there a more efficiently way of doing this query in SQLAlchemy?

**Comment by anticorrelator** [comment]
can you rebase this on top of datasets?

**Comment by axiomofjoy** [comment]
> can you rebase this on top of datasets?

Gonna squash merge, though?

File: src/phoenix/server/api/schema.py (hunk 1 above)
Comment (axiomofjoy) [reply src/phoenix/server/api/schema.py:505]: Going to switch to [this pattern](https://docs.sqlalchemy.org/en/20/changelog/whatsnew_20.html#bulk-insert-with-returning).

File: src/phoenix/server/api/schema.py [hunk 2]
```diff
@@ -384,6 +402,140 @@ async def create_dataset(
...
+
+INPUT_MIME_TYPE = SpanAttributes.INPUT_MIME_TYPE
+INPUT_VALUE = SpanAttributes.INPUT_VALUE
+OUTPUT_MIME_TYPE = SpanAttributes.OUTPUT_MIME_TYPE
+OUTPUT_VALUE = SpanAttributes.OUTPUT_VALUE
+LLM_PROMPT_TEMPLATE_VARIABLES = SpanAttributes.LLM_PROMPT_TEMPLATE_VARIABLES
+LLM_INPUT_MESSAGES = SpanAttributes.LLM_INPUT_MESSAGES
+LLM_OUTPUT_MESSAGES = SpanAttributes.LLM_OUTPUT_MESSAGES
+RETRIEVAL_DOCUMENTS = SpanAttributes.RETRIEVAL_DOCUMENTS
+
```
Comment (mikeldking) [inline src/phoenix/server/api/schema.py:538]: I'm guessing you will put this in it's own file as a follow-up

**Review by mikeldking — APPROVED** [review]

File: src/phoenix/server/api/schema.py (hunk 1 above)
Comment (axiomofjoy) [reply src/phoenix/server/api/schema.py]: addressed in [f72d4ee](https://github.com/Arize-ai/phoenix/pull/3205/commits/f72d4ee537434076c0495e8bbe845dabbc49a439)

**Comment by axiomofjoy** [comment]
Rebased.

**Pull request merged**